app/src/main/java/io/pslab/communication/ScienceLab.java

```diff
@@ -526,10 +532,25 @@ public void close() {
         }
     }
 
+    /**
+     * Blocking call that fetches an oscilloscope trace from the specified input channel
+     * @param channel Channel to select as input. ['CH1'..'CH3','SEN']
```

Comment:
List all the channels. This is the direct documentation for a developer to refer to.